classes/class-loader.php

```diff
@@ -633,6 +633,11 @@ public function set_color_palette() {
 					'slug'  => 'yellow',
 					'color' => '#ffd204',
 				],
+				[
```

Comment:
It looks like SonarCloud is complaining about these lines being duplicated, but I don't see a way around it 🤔

Reply:
We can now define color palette's in `theme.json`, I think it should have the same result as defining them here.

Reply:
https://developer.wordpress.org/themes/advanced-topics/theme-json/#preset-values

Reply:
Not sure what that means for translations, though.